### ISSUE TITLE
【EESS】New submissions for Mon, 19 Jun 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#318]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
## Keyword: medical imaging
There is no result 
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
### Neural Volumetric Reconstruction for Coherent Synthetic Aperture Sonar
 - **Authors:** Albert W. Reed, Juhyeon Kim, Thomas Blanford, Adithya Pediredla, Daniel C. Brown, Suren Jayasuriya
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2306.09909
 - **Pdf link:** https://arxiv.org/pdf/2306.09909
 - **Abstract**
 Synthetic aperture sonar (SAS) measures a scene from multiple views in order to increase the resolution of reconstructed imagery. Image reconstruction methods for SAS coherently combine measurements to focus acoustic energy onto the scene. However, image formation is typically under-constrained due to a limited number of measurements and bandlimited hardware, which limits the capabilities of existing reconstruction methods. To help meet these challenges, we design an analysis-by-synthesis optimization that leverages recent advances in neural rendering to perform coherent SAS imaging. Our optimization enables us to incorporate physics-based constraints and scene priors into the image formation process. We validate our method on simulation and experimental results captured in both air and water. We demonstrate both quantitatively and qualitatively that our method typically produces superior reconstructions than existing approaches. We share code and data for reproducibility.
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
## Keyword: transfer function
There is no result 
### Crowdsourcing and Evaluating Text-Based Audio Retrieval Relevances
 - **Authors:** Huang Xie, Khazar Khorrami, Okko Räsänen, Tuomas Virtanen
 - **Subjects:** Audio and Speech Processing (eess.AS); Sound (cs.SD)
 - **Arxiv link:** https://arxiv.org/abs/2306.09820
 - **Pdf link:** https://arxiv.org/pdf/2306.09820
 - **Abstract**
 This paper explores grading text-based audio retrieval relevances with crowdsourcing assessments. Given a free-form text (e.g., a caption) as a query, crowdworkers are asked to grade audio clips using numeric scores (between 0 and 100) to indicate their judgements of how much the sound content of an audio clip matches the text, where 0 indicates no content match at all and 100 indicates perfect content match. We integrate the crowdsourced relevances into training and evaluating text-based audio retrieval systems, and evaluate the effect of using them together with binary relevances from audio captioning. Conventionally, these binary relevances are defined by captioning-based audio-caption pairs, where being positive indicates that the caption describes the paired audio, and being negative applies to all other pairs. Experimental results indicate that there is no clear benefit from incorporating crowdsourced relevances alongside binary relevances when the crowdsourced relevances are binarized for contrastive learning. Conversely, the results suggest that using only binary relevances defined by captioning-based audio-caption pairs is sufficient for contrastive learning.
## Keyword: video retrieval
There is no result 
### MobileASR: A resource-aware on-device personalisation framework for  automatic speech recognition in mobile phones
 - **Authors:** Zitha Sasindran, Harsha Yelchuri, Pooja Rao, T. V. Prabhakar
 - **Subjects:** Audio and Speech Processing (eess.AS); Artificial Intelligence (cs.AI)
 - **Arxiv link:** https://arxiv.org/abs/2306.09384
 - **Pdf link:** https://arxiv.org/pdf/2306.09384
 - **Abstract**
 We describe a comprehensive methodology for developing user-voice personalised ASR models by effectively training models on mobile phones, allowing user data and models to be stored and used locally. To achieve this, we propose a resource-aware sub-model based training approach that considers the RAM, and battery capabilities of mobile phones. We also investigate the relationship between available resources and training time, highlighting the effectiveness of using sub-models in such scenarios. By taking into account the evaluation metric and battery constraints of the mobile phones, we are able to perform efficient training and halt the process accordingly. To simulate real users, we use speakers with various accents. The entire on-device training and evaluation framework was then tested on various mobile phones across brands. We show that fine-tuning the models and selecting the right hyperparameter values is a trade-off between the lowest achievable performance metric, on-device training time, and memory consumption. Overall, our methodology offers a comprehensive solution for developing personalized ASR models while leveraging the capabilities of mobile phones, and balancing the need for accuracy with resource constraints.
### Knowledge Distillation for Efficient Audio-Visual Video Captioning
 - **Authors:** Özkan Çaylı, Xubo Liu, Volkan Kılıç, Wenwu Wang
 - **Subjects:** Audio and Speech Processing (eess.AS)
 - **Arxiv link:** https://arxiv.org/abs/2306.09947
 - **Pdf link:** https://arxiv.org/pdf/2306.09947
 - **Abstract**
 Automatically describing audio-visual content with texts, namely video captioning, has received significant attention due to its potential applications across diverse fields. Deep neural networks are the dominant methods, offering state-of-the-art performance. However, these methods are often undeployable in low-power devices like smartphones due to the large size of the model parameters. In this paper, we propose to exploit simple pooling front-end and down-sampling algorithms with knowledge distillation for audio and visual attributes using a reduced number of audio-visual frames. With the help of knowledge distillation from the teacher model, our proposed method greatly reduces the redundant information in audio-visual streams without losing critical contexts for caption generation. Extensive experimental evaluations on the MSR-VTT dataset demonstrate that our proposed approach significantly reduces the inference time by about 80% with a small sacrifice (less than 0.02%) in captioning accuracy.
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### Krasovskii Passivity for Sampled-data Stabilization and Output Consensus
 - **Authors:** Yu Kawano, Alessio Moreschini, Michele Cucuzzella
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2306.09706
 - **Pdf link:** https://arxiv.org/pdf/2306.09706
 - **Abstract**
 In this paper, we establish the novel concept of Krasovskii passivity for sampled discrete-time nonlinear systems, enabling Krasovskii-passivity-based control design under sampling. We consider two separate control objectives: stabilization and output consensus, where the latter is studied under the presence of an unknown constant disturbance. Inspired by methodologies in the continuous-time case, we develop sampled-data control schemes for each control objective based on Krasovskii passivity. The proposed sampled discrete-time controllers are respectively validated through simulations on a DC microgrid of boost converters and a DC microgrid of buck converters whose continuous-time models and their implicit midpoint discretizations are Krasovskii passive in each time scale.
### Super-Resolution Radar Imaging with Sparse Arrays Using a Deep Neural  Network Trained with Enhanced Virtual Data
 - **Authors:** Christian Schuessler, Marcel Hoffmann, Martin Vossiek
 - **Subjects:** Signal Processing (eess.SP); Computer Vision and Pattern Recognition (cs.CV); Robotics (cs.RO)
 - **Arxiv link:** https://arxiv.org/abs/2306.09839
 - **Pdf link:** https://arxiv.org/pdf/2306.09839
 - **Abstract**
 This paper introduces a method based on a deep neural network (DNN) that is perfectly capable of processing radar data from extremely thinned radar apertures. The proposed DNN processing can provide both aliasing-free radar imaging and super-resolution. The results are validated by measuring the detection performance on realistic simulation data and by evaluating the Point-Spread-function (PSF) and the target-separation performance on measured point-like targets. Also, a qualitative evaluation of a typical automotive scene is conducted. It is shown that this approach can outperform state-of-the-art subspace algorithms and also other existing machine learning solutions. The presented results suggest that machine learning approaches trained with sufficiently sophisticated virtual input data are a very promising alternative to compressed sensing and subspace approaches in radar signal processing. The key to this performance is that the DNN is trained using realistic simulation data that perfectly mimic a given sparse antenna radar array hardware as the input. As ground truth, ultra-high resolution data from an enhanced virtual radar are simulated. Contrary to other work, the DNN utilizes the complete radar cube and not only the antenna channel information at certain range-Doppler detections. After training, the proposed DNN is capable of sidelobe- and ambiguity-free imaging. It simultaneously delivers nearly the same resolution and image quality as would be achieved with a fully occupied array.
### Knowledge Distillation for Efficient Audio-Visual Video Captioning
 - **Authors:** Özkan Çaylı, Xubo Liu, Volkan Kılıç, Wenwu Wang
 - **Subjects:** Audio and Speech Processing (eess.AS)
 - **Arxiv link:** https://arxiv.org/abs/2306.09947
 - **Pdf link:** https://arxiv.org/pdf/2306.09947
 - **Abstract**
 Automatically describing audio-visual content with texts, namely video captioning, has received significant attention due to its potential applications across diverse fields. Deep neural networks are the dominant methods, offering state-of-the-art performance. However, these methods are often undeployable in low-power devices like smartphones due to the large size of the model parameters. In this paper, we propose to exploit simple pooling front-end and down-sampling algorithms with knowledge distillation for audio and visual attributes using a reduced number of audio-visual frames. With the help of knowledge distillation from the teacher model, our proposed method greatly reduces the redundant information in audio-visual streams without losing critical contexts for caption generation. Extensive experimental evaluations on the MSR-VTT dataset demonstrate that our proposed approach significantly reduces the inference time by about 80% with a small sacrifice (less than 0.02%) in captioning accuracy.
